src/main/java/gov/dot/its/codehub/adminapi/business/ConfigurationServiceImpl.java

```diff
@@ -273,4 +273,39 @@
 		}
 	}
 
+	@Override
+	public ApiResponse<List<String>> categoryImages(HttpServletRequest request) {
+		logger.info("Request: category images");
+		final String RESPONSE_MSG = "Response: GET Category images. ";
+
+		ApiResponse<List<String>> apiResponse = new ApiResponse<>();
+		List<ApiError> errors = new ArrayList<>();
+
+		if (!headerUtils.validateToken(request.getHeader(CHTOKEN_KEY))) {
+			logger.warn(String.format(MESSAGE_TEMPLATE, RESPONSE_MSG, MESSAGE_INVALID_TOKEN));
```

Comment:
Just FYI I think you could use the logger formatting here too. Not sure if it works with the prebuilt strings but I know sonar doesn't usually like the use of String.format.

Something like this:
```
private static final String MESSAGE_TEMPLATE = "{} : {} "
//
logger.warn(MESSAGE_TEMPLATE,  RESPONSE_MSG, MESSAGE_INVALID_TOKEN);
```

Ref: http://slf4j.org/apidocs/org/slf4j/Logger.html#debug%28java.lang.String,%20java.lang.Object%5B%5D%29

But no need to change this IMO.